### DISCUSSION TITLE
Anonymous accounts in GitHub?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#34595] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi.

Things like Pastebin allow you to share a snippet of code without necessarily registering the account. 

#### idea
*Would it be possible to contribute with open source with some anonymous account?*

*Why?* 
- This allows for greater anonymity and security for users who wish to collaborate with GitHub without being associated with the GitHub company.

**Comment by jfullstackdev** [comment, marked as answer]
Pastebin, on the other hand, is full of spam messages and hackers and that does not equate with security too

**Reply by ghost** [reply]
Hi jdevstatic.

> Pastebin, on the other hand, is full of spam messages and hackers and that does not equate with security too

1. These problems are not with Pastebin, but with those who use Pastebin. For example, someone could misuse open source for some spam tool. But the problem is not whether the code is open or not, but who uses the tool.

2. This does not guarantee greater security, on the other hand, it guarantees greater privacy for the user. Which gives the user the feeling of having security and control of the data. Also, if open source, why would you need to have an account on any platform?

3. I believe that this problem you reported could be solved with a unique hash key, where one or more people are idenfified without compromising the security of data about the user. 

4. There are things like git-bug, radicle which utilizes the concept of "anonymity". For example, it is possible to synchronize several issues in different platforms, this guarantees security in case any platform stops working or if you don't want to feel tied to a specific platform. In addition to these two use cases, with git-bug you have anonymity in the sense of deciding which services you want to sync your issues, and of course aware of the term of use, privacy, security. It works the same way for radicle, which is a git-p2p rotocol, where you contribute in a decentralized network and distribute your open codes.

*I think your point of view is very valid, but I forgot to add this other information. But anyway, thanks for the feedback.*
